rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -536,9 +536,12 @@ def _query(self, method: Callable, call_order: Sequence[WeightedNode], **kwargs:
             return result
 
         # no node in the call order list was succesfully queried
-        raise RemoteError(
+        log.error(
             f'Failed to query {method!s} after trying the following '
-            f'nodes: {[str(x) for x in call_order]}. Check logs for details.',
+            f'nodes: {[str(x) for x in call_order]}',
+        )
+        raise RemoteError(
+            f'All connected nodes were queried and no response was returned'
```

Comment:
What I think Yabir is trying to say here is that it would be nice to add:
1. The EVM chain name
2. Perhaps add small note to the error that they should check their networking and that enough nodes are connected for the chain.
3. 
But keep the message short.

Reply:
Hey, Thanks for the help.

For user:
```
raise RemoteError(
            f'Please check your network and confirm sufficient nodes are connected for the blockchain to function correctly.')
 ```
 
 For log:
```
log.error(
            f'Failed to query {method!s} after trying the following '
            f'nodes: {[x.node_info.name for x in call_order]}',)
```
 Does that look good to you?